### PR TITLE
Adding announcement to landing page

### PR DESCRIPTION
**Pull request by shanzzam**
Adding new client announcement.
Fixes stellar/nebula/issues/169

### CODE REVIEW

**Comment by shanzzam** [comment]
@quietbits The link text is still a bit dark. Not sure the best way to adjust that w/ css on the TextLink. 
dw_announce

File: src/App.scss [hunk 1]
```diff
@@ -59,6 +59,14 @@ code {
   overflow: auto;
 }
 
+.Announcement {
+    background-color: var(--color-note);
+    color: var(--color-text-contrast);
+    text-align: center;
+    width: 100%;
+    margin-bottom: 1rem;
+}
```
Comment (quietbits) [inline src/App.scss:68]: To make the link the same color as the text, we need to add this inside the `.Announcement` class:

```css
.Announcement {
  ...
  padding: 1rem 0; // We might add vertical padding to let the text breathe a bit

  // This is the class of the anchor
  .TextLink {
    color: inherit;
  }
}
```

Spacing looks a bit off here (we use 2 spaces everywhere).

File: src/App.tsx [hunk 2]
```diff
@@ -38,6 +38,21 @@ export const App = () => (
...
+                  log bugs and feature requests at: &nbsp;
+                  <TextLink
+                    variant={TextLinkVariant.primary}
+                    href="https://github.com/stellar/stellar-demo-wallet/issues"
+                    rel="noreferrer"
+                    target="_blank"
+                  >
+                    https://github.com/stellar/stellar-demo-wallet/issues
+                  </TextLink>
+                </p>
```
Comment (quietbits) [inline src/App.tsx:53]: To have consistent width and horizontal padding, we need to wrap `<p>` in `<div className="Inset">`

**Review by quietbits — APPROVED** [review]
Great job! 🎉 Just left minor comments. 🙂

File: src/App.scss (hunk 1 above)
Comment (shanzzam) [reply src/App.scss:68]: @quietbits spacing on the padding?

**Pull request merged**